### PR TITLE
Fix/ Slug transliteration

### PR DESCRIPTION
**Pull request by obniavko**
dev
## ZeroWaste project

* [Project ticket #791](https://github.com/ita-social-projects/ZeroWaste/issues/791)

## Code reviewers

- [ ] @kisiohlova 
- [ ] @loqimean 

## Summary of issue

Transliteration of slugs is incorrect.

## Summary of change

1. Fixed transliteration.
2. Changed column order of Calculators table ('Name' to 'Slug' instead of 'Slug' to 'Name', because 'Name' has higher priority).

Slug

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

File: app/views/account/calculators/index.html.erb [hunk 1]
```diff
@@ -1,34 +1,37 @@
...
-
-      <span class="mx-2"></span>
-      <%= link_to new_account_calculator_path, class: "btn-green px-4 py-2" do %>
+  <% set_meta_tags(title: t(".title")) %>
+  <div class="row">
+    <div class="col-md-6">
+      <%= search_form_for(@q, url: [:account, :calculators], method: :get, class: "d-flex mb-5") do |f| %>
+        <%= f.search_field :name_cont, placeholder: t(".search_placeholder"), class: "form-input me-2" %>
+        <%= f.button t(".search_button"), class: "btn btn-blue px-4 items-center" %>
+      <% end %>
```
Comment (loqimean) [inline app/views/account/calculators/index.html.erb:8]: в сьорч форм паршал

**Review by kisiohlova — APPROVED** [review]

File: app/views/account/calculators/index.html.erb (hunk 1 above)
Comment (obniavko) [reply app/views/account/calculators/index.html.erb]: Fixed. Якщо все ок, то для існуючих сторінок Categories, Messages, Users винесу окремим пулреквестом.

File: app/views/account/shared/_search_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<div class="col-md-6">
+  <%= search_form_for(@q, url: search_url, method: :get, class: "d-flex mb-5") do |f| %>
```
Comment (loqimean) [inline app/views/account/shared/_search_form.html.erb:2]: знову інстанс змінна), не гуд

Comment (obniavko) [reply app/views/account/shared/_search_form.html.erb]: Fixed

File: app/views/account/calculators/index.html.erb [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
-      <%= search_form_for(@q, url: [:account, :calculators], method: :get, class: "d-flex mb-5") do |f| %>
-        <%= f.search_field :name_cont, placeholder: t(".search_placeholder"), class: "form-input me-2" %>
-        <%= f.button t(".search_button"), class: "btn btn-blue px-4 items-center" %>
-      <% end %>
-    </div>
+    <%= render partial: "account/shared/search_form", locals: {
+      q: @q,
+      search_url: account_calculators_path,
+      search_attribute: :name_cont
+    } %>
```
Comment (loqimean) [inline app/views/account/calculators/index.html.erb:8]:
```suggestion
<%= render partial: "account/shared/search_form", locals: {
      q: @q,
      search_url: account_calculators_path,
      search_attribute: :name_cont
    } %>
```

**Review by loqimean — APPROVED** [review]

**Pull request merged**